src/pylutron_caseta/smartbridge.py

```diff
@@ -900,12 +921,10 @@ async def _load_devices(self):
             )
 
     async def _load_ra3_devices(self):
+        coros: List[Coroutine[Any, Any, None]] = []
         for area in self.areas.values():
-            await self._load_ra3_control_stations(area)
-            await self._load_ra3_zones(area)
-
-        # caseta does this by default, but we need to do it manually for RA3
-        await self._subscribe_to_multi_zone_status()
+            coros.append(self._load_ra3_control_stations(area))
+            coros.append(self._load_ra3_zones(area))
```

Comment:
Are we missing a gather(*coros) after this?

Reply:
Yup I forgot to put it back when I moved the subscribe. Pushed now